src/app/components/addresses/utils/getAddressFromBagPdokResponse.ts

```diff
@@ -0,0 +1,6 @@
+export const getAddressFromBagPdokResponse = (data?: BAGPdokResponse):  BAGPdokAddress | undefined => {
+  const docs = data?.response?.docs
+  return (
+    docs?.find((result) => result.adrestype === "hoofdadres")
```

Comment:
Zijn checks op hoofdadres nog nodig? In de query naar de api filter je daar ook al op. Ik weet ook niet of ze zaken moeten kunnen aanmaken op een nevenadres?